LocalSettings.php

```diff
@@ -2975,6 +2975,81 @@
 			'amc' => true,
 		],
 	],
+	'wgMFQueryPropModules' => [
+		'default' => [ 
+			'pageprops', 
+		],
+		'gratisdatawiki' => [ 
+			'entityterms',
+		],
+	],
+	'wgMFSearchAPIParams' => [
+		'default' => [
+			'ppprop' => 'displaytitle',
+		],
+		'gratisdatawiki' => [
+			'wbetterms' => 'label',
+		],
+	],
+	'wgMFSearchGenerator' => [
+		'default' => [
+			'name' => 'prefixsearch',
+			'prefix' => 'ps',
+		],
+		'gratisdatawiki' => [
+			'name' => 'wbsearch',
+			'prefix' => 'wbs',
+		],
+	],
+	'wgMFVaryOnUA' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wgMFUseWikibase' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+		'gratisdatawiki' => true,
+	],
+	'wgMFEnableWikidataDescriptions' => [
+		'default' => [
+			'base' => false,
+			'beta' => true,
+		],
+		'gratispaideiawiki' => [
+			'base' => true,
+			'beta' => true,
+		],
+	],
+	'wgMFDisplayWikibaseDescriptions' => [
+		'default' => [
+			'search' => false,
+			'nearby' => false,
+			'watchlist' => false,
+			'tagline' => false,
+		],
+		'gratispaideiawiki' => [
+			'search' => true,
+			'nearby' => true,
+			'watchlist' => true,
+			'tagline' => false,
+		],
+		'gratisdatawiki' => [
+			'search' => true,
+			'nearby' => true,
+			'watchlist' => true,
+			'tagline' => false,
+		],
+	],
+	'wgMFNearby' => [
```

Comment:
I don't think we support this functionality. Even if we do, I'd be somewhat concerned with it as it geo locates users, so I have some privacy concerns here as well. Unless I'm not understanding it.